packages/api/src/services/comissariat.service.ts

```diff
@@ -0,0 +1,81 @@
+import { Context } from "../context";
+
+const EXPIRES_IN = 60 * 5;
+
+export const getAll = async (
+  ctx: Context,
+  limit: number,
+  cursor: number | undefined
+) => {
+  let items = await ctx.prisma.comissariat.findMany({
+    take: limit + 1,
+    cursor: cursor ? { id: cursor } : undefined,
+    orderBy: { id: "asc" },
+  });
+  let nextCursor: typeof cursor | undefined = undefined;
+  if (items.length > limit) {
+    const nextItem = items.pop();
+    nextCursor = nextItem?.id;
+  }
+  items = await Promise.all(
+    items.map(async (comissariat) => {
+      if (comissariat.logo) {
+        comissariat.logo = await ctx.storage.createSignedUrl(
```

Comment:
i think for logo we don't need to signed url, public is fine, since this is not secured information.
and it will result N+1 query.

but for except for uploaded document we will need signed url.